mobile/src/stateMachine/views/main/settings.tsx

```diff
@@ -58,6 +58,10 @@ export class SettingsView extends ViewState {
         this.trigger(ScenarioTriggers.Submit);
     }
 
+    private onHealthChange = () => {
```

Comment:
Similar to before, this should ideally be called `onHealthPermissionsChange` or something